.github/workflows/validation_benchmarks.yml

```
@@ -5,11 +5,17 @@ on:
     types:
       - published
 
+
+permissions:
+  contents: read
+
```

```suggestion

permissions: {}
```

Let's grant the workflow minimal permissions, and the jobs extended permissions.